docs/proposals/resource-catalog.md

```diff
@@ -0,0 +1,222 @@
+
+
+We can reshape the catalog resource to more easily express them, secure them, and claim them.
+
+# Goals
+ 1.  cluster-admin should control which catalog exist.
+ 2.  cluster-admin should be able/disable per-catalog exposure on a per-project granularity.
```

Comment:
Use case?  I don't have one off the top of my head - we generally only hard
prevent something if it is a security risk to everyone else to do that.

On Mon, Apr 25, 2016 at 11:28 AM, David Eads <notifications@github.com>
wrote:

> In docs/proposals/resource-catalog.md
> <https://github.com/ncdc/kubernetes/pull/8#discussion_r60933395>:
>
> > @@ -0,0 +1,222 @@
> > +
> > +
> > +We can reshape the catalog resource to more easily express them, secure them, and claim them.
> > +
> > +# Goals
> > + 1.  cluster-admin should control which catalog exist.
> > + 2.  cluster-admin should be able/disable per-catalog exposure on a per-project granularity.
>
> Why not role based, i.e. is per project more or less valuable than role?
>
> I think you'd want to prevent some projects from exposing resources even
> if a user has authority to do it.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ncdc/kubernetes/pull/8/files/a2acd970c6316ce196c7bb3f24fb57e1bbef647e#r60933395>
>